test/tests/load.js

```diff
@@ -64,4 +64,4 @@
 /*#*/ include('Numerical.js');
 
 // There is no need to test interactions in node context.
-if (!isNode) /*#*/ include('Interactions.js');
+/*#*/ if (!isNode) include('Interactions.js');
```

Comment:
@sasensi does this work? I don't think `isNode` is a pre-process-time global variable.

Could you try write this code like this, on three lines?

```js
if (!isNode) {
    /*#*/ include('Interactions.js');
}
```

Reply:
I don't understand, test now work on my local travis but still fail on online one !
I'll try your idea.

Reply:
@sapics also pointed out weirdness with Travis... I really have no clue what's going on.

Reply:
I think `/*#*/` needs to be on its own line, not nested with other code that is evaluated at execution time.

Reply:
I'll try to revert changes in a separated branch to see if failure is really related to code changes.